### PR TITLE
Remove quanto feed

### PR DESCRIPTION
**Pull request by tommyrharper**

### CODE REVIEW

File: markets/perps-market/contracts/modules/MarketConfigurationModule.sol [hunk 1]
```diff
@@ -108,6 +108,8 @@ contract MarketConfigurationModule is IMarketConfigurationModule {
         OwnableStorage.onlyOwner();
         PerpsMarketConfiguration.Data storage config = PerpsMarketConfiguration.load(marketId);
         config.quantoSynthMarketId = quantoSynthMarketId;
+        // TODO: test this event
```
Comment (JaredBorders) [inline markets/perps-market/contracts/modules/MarketConfigurationModule.sol:111]: remove this before merge please

File: markets/perps-market/contracts/storage/PerpsPrice.sol [hunk 2]
```diff
@@ -62,11 +58,23 @@ library PerpsPrice {
...
-        if (isQuanto && feedId == bytes32(0)) {
-            return 1 ether;
+
+        bytes32 feedId;
+        if (isQuanto) {
+            PerpsMarketConfiguration.Data storage config = PerpsMarketConfiguration.load(marketId);
+            uint128 quantoSynthMarketId = config.quantoSynthMarketId;
+
+            /// @dev if the quantoSynthMarketId is not set, the base asset is USD, which has a price of 1 USD per USD
+            // TODO: check 100% this always works
```
Comment (JaredBorders) [inline markets/perps-market/contracts/storage/PerpsPrice.sol:68]: check this before merge

File: markets/perps-market/contracts/storage/PerpsPrice.sol [hunk 3]
```diff
@@ -62,11 +58,23 @@ library PerpsPrice {
...
+            if (quantoSynthMarketId == 0) {
+                return 1 ether;
+            }
+
+            // TODO: write comment explaining why this is sellFeedId
```
Comment (JaredBorders) [inline markets/perps-market/contracts/storage/PerpsPrice.sol:73]: please do this before merge

**Review by JaredBorders — APPROVED** [review]
LGTM, just please address the `TODO` comments that are still hanging around

File: markets/perps-market/contracts/storage/PerpsPrice.sol (hunk 2 above)
Comment (tommyrharper) [reply markets/perps-market/contracts/storage/PerpsPrice.sol]: Done.

File: markets/perps-market/contracts/storage/PerpsPrice.sol (hunk 3 above)
Comment (tommyrharper) [reply markets/perps-market/contracts/storage/PerpsPrice.sol]: Done.

**Pull request merged**